src/model/stl/memory_impl.rs

```diff
@@ -1,5 +1,6 @@
-use std::ops::{Deref, DerefMut};
```

Comment:
rustfmt changes

Reply:
These changes come from building with the `generate_model_apis` feature on.